packages/calypso-build/webpack/transpile.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Return a Webpack loader configuration object containing for JavaScript transpilation.
+ *
+ * @param {Object} _                 Options
+ * @param {number} _.workerCount     Number of workers that are being used by the thread-loader
+ * @param {string} _.configFile      Babel loader config file
+ * @param {string} _.cacheDirectory  Babel loader cache directory
+ * @param {string} _.cacheIdentifier Babel loader cache identifier
```

Comment:
I'd remove the word `loader` from these three param descriptions -- these things aren't really specific to the Babel _loader_ but to Babel more generally.

Reply:
Made the change in 05c541c237.